backend/src/main/java/com/funeat/product/domain/Product.java

```diff
@@ -49,6 +49,11 @@ public Product(final String name, final Long price, final String image, final St
         this.category = category;
     }
 
+    public void updateAverageRating(final Long rating, final Long count) {
+        double calculatedRating = ((count - 1) * averageRating + rating) / count;
+        this.averageRating = Math.round(calculatedRating * 10.0) / 10.0;
+    }
+
```

Comment:
```suggestion
    public void updateAverageRating(final Long rating, final Long count) {
        final double calculatedRating = ((count - 1) * averageRating + rating) / count;
        this.averageRating = Math.round(calculatedRating * 10.0) / 10.0;
    }
```

10.0도 상수로 해두는 것도 좋아보입니다~

Reply:
count = 0인 상황이 존재하지 않게 리뷰 먼저 등록하고, 평균 평점을 업데이트하는 것도 좋네요 👍